spring-cloud-bus/src/main/java/org/springframework/cloud/bus/jackson/BusJacksonAutoConfiguration.java

```diff
@@ -39,23 +39,20 @@
 public class BusJacksonAutoConfiguration {
 
 	@Bean
+	@ConditionalOnMissingBean(name="busJsonConverter")
```

Comment:
@spencergibb I took that from [this commit](https://github.com/spring-cloud/spring-cloud-bus/blob/bef97a523c206cd583e02691c817076a93b026b3/spring-cloud-bus/src/main/java/org/springframework/cloud/bus/jackson/BusJacksonAutoConfiguration.java#L44) but happy to change if the name is not necessary